tests/features/rhpam/kieserver/rhpam-kieserver.feature

```diff
@@ -17,13 +17,14 @@ Feature: RHPAM KIE Server configuration tests
       | CUSTOM_INSTALL_DIRECTORIES | custom   |
     Then file /opt/eap/standalone/deployments/node-info.war should exist
 
-  # KIECLOUD-11: temporarily ignore since Jenkins CI builds currently fail with this test, even though it passes when run at the command line
-  @ignore
   Scenario: deploys the library example, then checks if it's deployed.
-    Given s2i build https://github.com/jboss-container-images/rhpam-7-openshift-image from quickstarts/library-process/library using rhpam71-dev
+    Given s2i build https://github.com/jboss-container-images/rhpam-7-openshift-image from quickstarts/library-process/library using master
       | variable                        | value                                                                                    |
-      | KIE_SERVER_CONTAINER_DEPLOYMENT | rhpam-kieserver-library=org.openshift.quickstarts:rhpam-kieserver-library:1.4.0-SNAPSHOT |
-    Then container log should contain Container rhpam-kieserver-library
+      | KIE_SERVER_CONTAINER_DEPLOYMENT | rhpam-kieserver-library=org.openshift.quickstarts:rhpam-kieserver-library:1.6.0-SNAPSHOT |
+      | JAVA_OPTS_APPEND                | -Djavax.net.ssl.trustStore=truststore.ts -Djavax.net.ssl.trustStorePassword=123456       |
+      | SCRIPT_DEBUG                    | true                                                                                     |
+    Then s2i build log should contain Attempting to verify kie server containers with 'java org.kie.server.services.impl.KieServerContainerVerifier  org.openshift.quickstarts:rhpam-kieserver-library:1.6.0-SNAPSHOT'
+    And s2i build log should contain java -Djavax.net.ssl.trustStore=truststore.ts -Djavax.net.ssl.trustStorePassword=123456 --add-modules
```

Comment:
--add-modules isn't present in the JAVA_OPTS_APPEND value, could be removed

Reply:
not necessary.
Enable the debug, set the java opts append env and inspect the logs, that line is printed during container verify step.

Reply:
Is the verification on the entire line and not on parts of it ? If is on the entire line is ok to leave as is.

Reply:
Is the verification on the entire line and not on sub parts of it ? If is on the entire line is it ok to leave as is.